source/macos/AdaptiveCards/AdaptiveCards/Views/ACRMultilineInputTextView.swift

```diff
@@ -128,18 +174,22 @@ extension ACRMultilineInputTextView: InputHandlingViewProtocol {
     }
     
     var isValid: Bool {
-        return true
+        return !hasError
     }
 }
 
 extension ACRMultilineInputTextView: ACRTextViewResponderDelegate {
     func textViewDidBecomeFirstResponder() {
         scrollView.disableScroll = false
-        layer?.borderColor = inputConfig.activeBorderColor.cgColor
+        if !textFieldShowsError {
```

Comment:
textFieldShowsError is used everywhere quite a lot. 
Not really comfortable of having it a `public get set` property. Would be better if these logics are handled without any flags like `textFieldShowsError`.
Should go crazily wrong if this flag is not updated correctly or goes outdated somewhere. Would be better if rely on live values.